griptape/tools/google_gmail/tool.py

```diff
@@ -0,0 +1,71 @@
+import logging
+import base64
+from email.message import EmailMessage
+from schema import Schema, Literal
+from attr import define
+from griptape.artifacts import TextArtifact, ErrorArtifact
+from griptape.core.decorators import activity
+from griptape.tools import BaseGoogleClient
+
+
+@define
+class GoogleGmailClient(BaseGoogleClient):
+
+    @activity(config={
+        "description": "Can be used to create a draft email in GMail",
+        "schema": Schema({
+            Literal(
+                "to",
+                description="email address which to send to"
+            ): str,
+            Literal(
+                "subject",
+                description="subject of the email"
+            ): str,
+            Literal(
+                "from",
+                description="email address which to send from"
+            ): str,
+            Literal(
+                "body",
+                description="body of the email"
```

Comment:
Later we can update it to include an email body from memory as an alternative to passing it directly.